src/components/Login.js

```diff
@@ -1,10 +1,53 @@
-import React from "react"
+import React, { useState } from "react"
+import { navigate } from "@reach/router"
+import GoogleButton from 'react-google-button'
+import { googleProvider } from "../config/authMethod"
+import { loginAuth } from "../service/auth"
+import { logoutAuth } from "../service/auth"
 
-const Login = () => (
-  <div className="loginbox">
-    Let's create your CV!
-    <button className="btn btn--orange">Login with Incentro account</button>
-  </div>
-)
 
-export default Login
\ No newline at end of file
+//TODO:
+//Change elements copied fror the index to work in a functional component
+//Add correct imports
+//Style the login page
+//Add redirect after login / check for autoredirect after login
+
+const Login = () => {
+  const [isSignedIn, setIsSignedIn] = useState(false)
+
+  const handleLogin = async provider => {
+    const res = await loginAuth(provider)
+    console.log(res)
+    if (res) {
+      setIsSignedIn(!isSignedIn)
+      navigate('./privatetest')
+    }
+    // console.log(res)
+    // console.log(isSignedIn)
+  }
+
+  const handleLogout = async () => {
+    setIsSignedIn(!isSignedIn)
+    const res = await logoutAuth()
+    // console.log(res)
+  }
+
+  const ifUserSignedIn = () => {
+    if (isSignedIn === null) {
+      return <h1>Eens kijken of je ingelogt bent...</h1>
+    }
+    return isSignedIn ? (
+      <GoogleButton onClick={() => handleLogout()}>ingelogd</GoogleButton>
```

Comment:
dit lijkt een log out button, willen we daar niet iets al "uiloggen" in zetten?

Reply:
Opgelost